tests/modeles/magasin.js

```diff
@@ -8,10 +8,10 @@ describe('Le magasin', function () {
       unContenantUnitaire('Premium Terra', 3)
     ).construit();
 
-    expect(magasin.produitsEnStock()).to.eql([
-      { nom: 'Nova Sky', type: 'ContenantUnitaire', quantite: 12 },
-      { nom: 'Premium Terra', type: 'ContenantUnitaire', quantite: 3 }
-    ]);
+    let produits = magasin.produitsEnStock();
```

Comment:
C'est étrange qu'on soit revenue à cette forme de test "éclaté" ? commentaire valable aussi pour les deux tests suivants

Reply:
`magasin.produitsEnStock()` renvoie maintenant une `Map` (et non plus un `Array`), et je n'avais pas d'idée simple pour comparer cette `Map` retournée avec une `Map` attendue. (`expect(map).to.eql(autreMap)` ne donne pas de résultat fiable, j'aurai besoin de regarder pourquoi plus en détail).

En y regardant à deux fois, je me dis qu'on peut passer par la comparaison d'un `Array.from(map)` avec un `Array` attendu. J'ai modifié le code en ce sens.